docs/tutorials/04_encrypted_ebs.rst

```diff
@@ -3,12 +3,12 @@
 .. toctree::
    :maxdepth: 2
 
-###################################
-Encrypted EBS with a Custom KMS Key
-###################################
+#####################################
+Disk Encryption with a Custom KMS Key
+#####################################
 
-AWS ParallelCluster supports the configuration option ``ebs_kms_key_id``, which allows you to provide a custom KMS key
-for EBS Disk encryption, to use it you'll need to specify a ``ec2_iam_role``.
+AWS ParallelCluster supports the configuration option ``ebs_kms_key_id`` and ``fsx_kms_key_id``, which allows you to
```

Comment:
configuration options...which allow...custom KMS key for EBS Disk (FSx is missing)

Reply:
Added